### PR TITLE
Added some integration tests for controllers

### LINKED ISSUE #54: Desarrollar tests de integración

### PR DESCRIPTION
**Pull request by noloman**
fixes #54

### CODE REVIEW

File: test/controllers/meetups_controller_test.rb [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
-  # end
+  test 'should show main page' do
+    get '/meetups/index'
+    assert_response :redirect
+    follow_redirect!
+    assert_response :success
+  end
+
+  test 'should redirect to meetup page if there is not id in params' do
+    get new_meetup_path
```
Comment (alvarocasadoc) [inline test/controllers/meetups_controller_test.rb]: Actually, you don't have to provide an id to go to `new_meetup_path`. The only requirement is that there's a logged in user, this means that `current_user` exists (we use [devise](https://github.com/plataformatec/devise)).

If the user has not logged in, he/she will be redirected to the `meetups_path`.

This validation is made through the [meetups policy](https://github.com/infusionvlc/infusion/blob/master/app/policies/meetup_policy.rb) thanks to the [pundit gem](https://github.com/varvet/pundit).

**Comment by alvarocasadoc** [comment]
Really well done! Thank you again for your help handling controller testing. If you don't have any more concerns with this PR I think we can merge your first tests into the project :100: :rocket:

**Comment by noloman** [comment]
hey! 
I was thinking of maybe creating some more tests, for other controllers and once it is done then merge them, or should I create other PR for the other controller tests?

**Comment by alvarocasadoc** [comment]
Feel free to do it whatever way you feel more comfortable with. As long as we can review the code every few commits it won't be a problem 👍

**Comment by noloman** [comment]
> Feel free to do it whatever way you feel more comfortable with. As long as we can review the code every few commits it won't be a problem 👍

Ok, then I will just merge this one and will create a different PR for a different controller test file! Thanks!

**Pull request merged**